app/src/main/res/values-id/strings.xml

```diff
@@ -0,0 +1,373 @@
+    <string name="layout_deleted">Tata letak dihapus</string>
+
+    <string name="layout_editor">Editor tata letak</string>
+    <string name="add_component">Tambahkan komponen</string>
+    <string name="no_more_components">Tidak ada lagi komponen untuk ditambahkan</string>
+    <string name="label_size">Ukuran:</string>
+    <string name="menu">Menu</string>
+    <string name="properties">Properti</string>
+    <string name="revert_changes">Kembalikan perubahan</string>
+    <string name="reset_default">Setel ulang ke default</string>
+    <string name="exit_without_saving">Keluar tanpa menyimpan</string>
+    <string name="portrait_background">Latar belakang potret</string>
+    <string name="portrait_background_mode">Mode latar belakang potret</string>
+    <string name="landscape_background">Latar belakang lanskap</string>
+    <string name="landscape_background_mode">Mode latar belakang lanskap</string>
+    <string name="save_and_exit">Simpan dan keluar</string>
+    <string name="layout_name">Nama tata letak</string>
+    <string name="layout_orientation">Orientasi tata letak</string>
+    <string name="layout_use_default_opacity">Gunakan transparasi default</string>
+    <string name="layout_opacity">Transparasi tata letak</string>
+    <string name="layout_background_load_failed">Gagal memuat latar belakang</string>
+
+    <string name="backgrounds">Latar belakang</string>
+    <string name="action_backgrounds_new">Latar belakang baru</string>
+    <string name="background_name">Nama latar belakang</string>
+    <string name="background_deleted">Latar belakang dihapus</string>
+    <string name="background_add_wrong_orientation">Latar belakang ditambahkan tetapi tidak terlihat untuk orientasi yang dipilih.</string>
+    <string name="background_add_processing_failed">Gagal memproses gambar latar yang dipilih. Pastikan tidak rusak.</string>
+
+    <string-array name="theme_options" tools:ignore="InconsistentArrays">
+        <item>@string/theme_option_light</item>
+        <item>@string/theme_option_dark</item>
+    </string-array>
+
+    <string-array name="fast_forward_speed_multiplier_options" tools:ignore="InconsistentArrays">
+        <item>Unlimited</item>
+        <item>1.5x</item>
+        <item>2x</item>
+        <item>3x</item>
+        <item>4x</item>
+        <item>8x</item>
+    </string-array>
+
+    <string-array name="rom_icon_filtering_options">
+        <item>None (Pixelated)</item>
+        <item>Linear (Halus)</item>
+    </string-array>
+
+    <string-array name="game_runtime_console_type_options">
+        <item>Default</item>
+        <item>@string/console_ds</item>
+        <item>@string/console_dsi</item>
+    </string-array>
+
+    <string-array name="firmware_settings_language_options">
+        <item>Inggris</item>
+        <item>Francis</item>
+        <item>Jerman</item>
+        <item>Italia</item>
+        <item>Spanyol</item>
+        <item>Jepang</item>
+        <item>Bahasa Indonesia</item>
```

Comment:
This extra option should not be present. These are the languages that the DS firmware was available on. Since it did not support Bahasa Indonesia, you should not add it.

Reply:
Ah yeah, sorry, and thanks:)
I'll be pay attention to an other changes in this language. Maybe not very often, but I'll check if anyone makes changes.(i'm not very good at english tbh, sorry)